### PR TITLE
WebPreview: Center URL in toolbar

### PR DESCRIPTION
**Pull request by spen**
### Summary
Following functional changes in #16329 and some of the concerns raised about how the URL is displayed, this PR aims to make visual improvements to the component.

### Test Plan
- Go to `/posts/{ your-mapped-domain }`
- Click edit to edit one of your posts
- Click the 'Update' button, wait for the preview to load.
- Have a :eyes: at the URL bar at different screen widths (note: you'll have to refresh the page at certain widths to show or hide different elements in the toolbar)

### CODE REVIEW

**Comment by folletto** [comment]
Could you please add before/after screenshots? Thanks!

**Comment by spen** [comment]
Good shout!

Here's a few at different BPs, you'll notice that on smaller screens the URL appears very much the same. There's an issue at widths just over 480px where they have very limited room because of the device dropdown - happy to tackle that here or in another PR:

#### 480
Before:
before-480

After:
after-480

#### 500
Before:
before-500

After:
after-500


#### Wide screens (I was less consistent with widths here, but they show the issues):
Before with short URL (appears off-center):
before-1660-shorter-url

Before with longer URL (appears centred, but clips early):
before-1245-long-url

After (URL can grow to fit most of the white space):
after-1044

**Comment by shaunandrews** [comment]
Looking good. Just two very minor things:

The input's :focus border color is being overridden by the :hover. Inputs normally have a dark border when focused, but in this case the lighter gray is over riding it when hovered and focused:

url input border on hover

I'd imagine this is a specificity issue related to the ordering of :focus and :hover in the CSS.

Second thing is that the input's text stays selected after "unfocusing" the input:

url sticky selection

The only way to unselect the text is to click on an inactive area of the toolbar or masterbar; Selecting text in the preview iframe gives you multiple selections on the page, which is confusing.

**Comment by marekhrabe** [comment]
I also noticed the thing with clicking the site and leaving the input selected. I investigated it a little bit and found that it would be quite hard to fix from a technical point of view because the input element doesn't actually lose the focus. What loses the focus is the whole Calypso window (just like when you open Finder on your computer to unfocus your browser) and the iframe window gets the focused. It's essentially two separate pages and each of those has its own focused element.

tldr: Calypso has a different focus than the iframe (the frontend of site) and there is no easy way to make them sync

**Comment by spen** [comment]
I'd given it some thought too and the best I came up with was blurring on some interaction (mouse out, timeout or something similar) but I feel that could prove tricky and come with it's own gotchas.
We might be safe to assume that once the 'Copy' button is clicked that we could blur the input then at least.

**Comment by marekhrabe** [comment]
Right after I posted my reply I think I've found a solution. It will be a separate DIFF + PR in few minutes, feel free to omit it from this one and only address the rest :D

**Comment by spen** [comment]
Awesome - I'm keen to do that since it'll be beyond just style changes.
The slight border color change on hover & focus is now fixed. Well spotted @shaunandrews, that one was subtle! 😃

**Comment by marekhrabe** [comment]
Are we good to go, @shaunandrews? Border seems to be consistent now and focus thingy is ready for review in #16547

**Comment by shaunandrews** [comment]
Looking good. Only last comment is the color of the URL — can we make it a few shades darker? Maybe use the same gray that's used on the URL in the sitecard:

screen shot 2017-07-25 at 4 26 42 pm

**Comment by spen** [comment]
Done - I think that looks a bit more prominent :)

**Comment by spen** [comment]
Thanks for your help with this guys <3
